lib/transaction_test.go

```diff
@@ -121,3 +120,34 @@ func TestTransactionIsValidCheckpoint(t *testing.T) {
 		return
 	}
 }
+
+func TestTransactionMaxOperationsInTransaction(t *testing.T) {
+	var err error
+
+	// well-formed
```

Comment:
I always confused about this `well-formed` sign. 
If anyone could give me some understand this `well-formed` mean to be? 
it point that well marshaled? or message itself well formatted?

Reply:
Well formed means passing contextless validation (e.g. doesn't have too many transaction, has valid address, amount, signatures...)
Checking things like account balance and so on require context so they are not included.
This difference is important because a transaction which is `well-formed` can be invalid and then valid (or the other way around), but a transaction which is not `well-formed` can never be valid and thus can be discarded immediately.

Reply:
@cmcm2222 Please see https://github.com/bosnet/sebak/issues/201#issuecomment-411670432 .

Reply:
@spikeekips @Geod24  thank you for kindness. :)